src/sdk/SpeakerIdentificationModel.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import { Contracts } from "./Contracts";
+import {
+    VoiceProfile,
+    VoiceProfileType,
+} from "./Exports";
+
+export class SpeakerIdentificationModel {
+    private privVoiceProfiles: VoiceProfile[] = [];
+
+    private constructor(profiles: VoiceProfile[]) {
+        Contracts.throwIfNullOrUndefined(profiles, "VoiceProfiles");
+        if (profiles.length === 0) {
+            throw new Error("Empty Voice Profiles array");
+        }
+        profiles.forEach((profile: VoiceProfile) => {
+            if (profile.profileType !== VoiceProfileType.TextIndependentIdentification) {
+                throw new Error("Identification model cannot be created from Verification profile: " + profile.profileId);
```

Comment:
>Verification [](start = 77, length = 12)

I'd rephrase as "Can only be made from...." because if another profile type appears the message will be misleading.

Reply:
Changed.